pool/app/admin/repo.ml

```diff
@@ -1,36 +1,82 @@
 module Dynparam = Utils.Database.Dynparam
 
 module Sql = struct
-  let select_from_users_sql ?order_by where_fragment =
-    let select_from =
+  let insert_request =
+    let open Caqti_request.Infix in
+    {sql|
+      INSERT INTO pool_admins (
+        user_uuid,
+        import_pending,
+        created_at,
+        updated_at
+      ) VALUES (
+        UNHEX(REPLACE($1, '-', '')),
+        $2,
+        NOW(),
```

Comment:
`created at` and `updated at` do have a default insert value, so they can be removed here.